examples/h2o/random_forest.ipynb

```diff
@@ -94,12 +94,13 @@
     "import mlflow\n",
     "import mlflow.h2o\n",
     "from h2o.estimators.random_forest import H2ORandomForestEstimator\n",
+    "\n",
     "h2o.init()\n",
     "\n",
     "wine = h2o.import_file(path=\"wine-quality.csv\")\n",
-    "r = wine['quality'].runif()\n",
```

Comment:
I never touch these notebooks, but `black .` formats them and create diff on my local machine where jupyter is installed.